coriolis/tests/osmorphing/test_suse.py

```diff
@@ -0,0 +1,376 @@
+# Copyright 2024 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+import logging
+from unittest import mock
+
+from coriolis import exception
+from coriolis.osmorphing import base
+from coriolis.osmorphing import suse
+from coriolis.tests import test_base
+
+
+class BaseSUSEMorphingToolsTestCase(test_base.CoriolisBaseTestCase):
+    """Test suite for the BaseSUSEMorphingTools class."""
+
+    def setUp(self):
+        super(BaseSUSEMorphingToolsTestCase, self).setUp()
+        self.event_manager = mock.MagicMock()
+        self.detected_os_info = {
+            'os_type': 'linux',
+            "distribution_name": suse.SLES_DISTRO_IDENTIFIER,
+            "release_version": "12",
+            'friendly_release_name': mock.sentinel.friendly_release_name,
+            'suse_release_name': 'test release'
+        }
+        self.package_names = ['package1', 'package2']
+        self.morphing_tools = suse.BaseSUSEMorphingTools(
+            mock.sentinel.conn, mock.sentinel.os_root_dir,
+            mock.sentinel.os_root_dir, mock.sentinel.hypervisor,
+            self.event_manager, self.detected_os_info,
+            mock.sentinel.osmorphing_parameters,
+            mock.sentinel.operation_timeout)
+
+    @mock.patch.object(
+        base.BaseLinuxOSMorphingTools, 'get_required_detected_os_info_fields'
```

Comment:
You don't need to mock anything for this test case. Just do assertions based on the actual return value of:
```python
suse.BaseSUSEMorphingTools.get_required_detected_os_info_fields()
```